dla_cnn/desi/DesiMock.py

```diff
@@ -109,7 +110,13 @@ def get_data(start_point = 0, end_point = self.data_size):
             sightline.dec = self.data[id]['DEC']
             sightline.dlas = self.data[id]['DLAS']
             sightline.loglam = np.log10(self.wavelength[start_point:end_point])
-
+            
+            if _normalize:
+                central = max(3800/(1+sightline.z_qso),1070)
```

Comment:
If `central == 1150`  (as it will for some), then 
you will be measuring the flux from 1150-1250 
not 1150-1170.  This is a bug.

Reply:
Also, let's make normalize its own method in `preprocess.py`

That way we can even have multiple options.  And this is a 
preprocessing step anyhow.  Not I/O

Reply:
I have discussed with @caiz12 , and we chose to drop those spectra whose `central>1170`, so we use 1170 as the upper limit for the slice. That's how we deal with this bug.
And I have removed the `_nomalize` method to the `preprocess.py`, but I also remains the parameter `nomalize` in `get_sightline`, I think it may be convenient in practice.